editor/limbo_ai_editor_plugin.cpp

```diff
@@ -914,6 +914,13 @@ void LimboAIEditor::_on_resources_reload(const PackedStringArray &p_resources) {
 #endif
 }
 
+void LimboAIEditor::_on_filesystem_changed() {
```

Comment:
Feels like an overkill to do it when LimboAI is not used.
I suggest connecting & disconnecting the signal in `make_visible`, or checking `is_visible_in_tree()`.

Reply:
It will introduce a bug that the tabs remain outdated when the editor is visible again.
I pushed a solution to update only when visible.